containers/ecr-viewer/tests/integration/listEcrDataService/core.test.ts

```diff
@@ -471,21 +471,16 @@ describe("generate filter conditions statement", () => {
     expect(params).toStrictEqual(["%Condition1%", "%Condition2%"]);
   });
 
-  it("should only look for eCRs with no conditions when de-selecting all conditions on filter", () => {
+  it("should return FALSE when de-selecting all conditions on filter", () => {
```

Comment:
can you add some integration tests to make sure the condition listing and filtering works as expected?